index.html

```diff
@@ -12,25 +12,30 @@
 </head>
 <body>
     <div class="menu">
-        <div class="bigsun"><img src="icons/bigsun.png" alt=""></div>
-        <h1>Your forecast app</h1>
-        <div class="search">
-            <span class="enter">enter your city
-                <div class="search__icon"><img src="icons/search.png" alt=""></div>
-            </span>
+        <div class="block__menu">
+            <div class="bigsun"><img src="icons/bigsun.png" alt=""></div>
+            <h1 class="title__one">Your forecast app</h1>
+            <div>
+                <div>
+                    <form name="search" method="post">
+                        <input class="enter" type="text" value="enter your city">
+                        <div class="search__icon"><a href="#"><img src="icons/search.png" alt=""></a></div>
```

Comment:
Better to change it to the `button` component. Or `input.submit` if you decided to use `form` here